Source/santactl/Commands/SNTCommandFileInfo.m

```diff
@@ -483,8 +483,9 @@ - (void)recurseAtPath:(NSString *)path {
   operationQueue.qualityOfService = NSQualityOfServiceUserInitiated;
 
   if (isDir && self.recursive) {
-    NSDirectoryEnumerator<NSString *> * dirEnum = [fm enumeratorAtPath:path];
-    for (NSString *file in dirEnum) {
+    NSDirectoryEnumerator *dirEnum = [fm enumeratorAtPath:path];
+    NSString *file = [dirEnum nextObject];
+    while (file) {
```

Comment:
The assignment needs to be inside the autorelease pool.  I did try doing more iterations inside the pool before it gets drained to lessen the overhead of having to create and destroy a pool each loop just to catch two strings, but it didn't seem to change much.